### PR TITLE
[14.0][ADD]account_reconciliation_widget_limit_aml: Filter all account move lines in reconciliation view

### PR DESCRIPTION
**Pull request by manuelregidor**
This module adds the option to filter all the account move lines by date in the reconcilation view, not only those coming from payments.

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Comment by HaraldPanten** [comment]
Related and coming from --> https://github.com/OCA/account-reconcile/pull/583

File: account_reconciliation_widget_limit_aml/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds the option to filter all the account move lines by date in the reconcilation view, not only those coming from payments.
```
Comment (HaraldPanten) [inline account_reconciliation_widget_limit_aml/readme/DESCRIPTION.rst]:
```suggestion
This module adds the option to filter all the account move lines by date in the reconciliation view, not only those coming from payments.
```

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM

**Comment by HaraldPanten** [comment]
@etobella do you mind having a look at this one?

@pedrobaeza I'm sure that you'll love it 😝

**Review by etobella — APPROVED** [review]

**Pull request merged**